auth/store.go

```diff
@@ -183,6 +186,9 @@ type AuthStore interface {
 
 	// HasRole checks that user has role
 	HasRole(user, role string) bool
+
+	//SetConsistentIndexSyncer sets consistentIndex syncer
```

Comment:
Need a space after "//" to fix goword checking: https://travis-ci.com/etcd-io/etcd/jobs/291835945#L409

Reply:
fixed. thanks